<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryManager</class>
 <widget class="QMainWindow" name="MemoryManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MemoryManager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>114</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="pageTableWidget">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>161</width>
      <height>271</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>Frame 0</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>Frame 7</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Physical Memory</string>
     </property>
    </column>
   </widget>
   <widget class="QPlainTextEdit" name="inputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>104</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputTextEdit">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>104</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>438</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionQuit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quito</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
